src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,88 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String createReportFull = readFromFile(fromFileName);
+        String createReport = createReport(createReportFull);
+        writeToFile(toFileName, createReport);
+    }
+
+    public String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            int value = bufferedReader.read();
+            while (value != -1) {
+                stringBuilder.append((char) value);
+                value = bufferedReader.read();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file1", e);
+        }
+        return stringBuilder.toString();
+    }
+
+    public String createReport(String createReportFull) {
+
+        String operationSupply = "supply";
+        String operationBuy = "buy";
+        int summaSupply = 0;
+        int summaBuy = 0;
+        int summaResult;
+
+        String[] arrayFromFileName = createReportFull.split("\n");
+        for (String record : arrayFromFileName) {
+            if (record.split(",")[0].equals(operationSupply)) {
+                summaSupply += parseInt(record.split(",")[1]);
+            }
+            if (record.split(",")[0].equals(operationBuy)) {
+                summaBuy += parseInt(record.split(",")[1]);
+            }
+        }
```

Comment:
move 0 and 1 to constants;
split array first and then use 'equals'
```suggestion
        for (String record : arrayFromFileName) {
            String operation = record.split(",")[OPERATION_INDEX];
            String amount = parseInt(record.split(",")[AMOUNT_INDEX]);
            ...
        }
```